index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+          <input type="text" name="name" class="input" placeholder="Full name">
+          <input type="email" name="email" class="input" placeholder="Email address">
+          <textarea class="textarea" name="message" placeholder="Enter text here"></textarea>
+          <button type="submit" class="contact-button">Get in touch</button>
+        </div>
+      </div>
```

Comment:
- [ ] Let's add the `<footer>` that we can see in the Figma template. This will complete the mobile design of your website, and you'll be set to start working on the desktop design.
![image](https://user-images.githubusercontent.com/79114233/135927660-6f8e25d2-0207-4769-b6a0-b14b732da062.png)